src/site/content/en/blog/how-we-built-designcember/index.md

```diff
@@ -0,0 +1,283 @@
+
+## Squooshing the art
+
+To maintain the fidelity of the illustration and ensure high definition screens wouldn't get a blurry user experience, Alice worked at a 3x pixel ratio. The plan was to use imgix and serve optimized images and formats on their server, but we found that manual tweaking with the Squoosh tool could save us 50% or more.
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/MSHzLyoiZdAZyy0oVHwo.png", alt="Using Squoosh to compress images.", width="800", height="490" %}
+
+Illustration has unique challenges for compression, especially the brush stroke and transparent rough edge style Alice used. We chose to Squoosh each 3x Photoshop exported png image, to a smaller png, webp, and avif. Each file type has its own special compression abilities, and it took compressing more than 50 images to find some common optimization settings.
+
+The [Squoosh CLI](https://github.com/GoogleChromeLabs/squoosh/tree/dev/cli) became crucial with over 200 images to optimize, doing all those by hand would take days. Once we had the common optimization settings, we were able to provide them as command line instructions and batch process entire folders of PNG images into their WebP and AVIF compressed counterparts. 
+
+Here's an example AVIF CLI squoosh command used:
+
+```bash
+npx @squoosh/cli --quant '{"enabled":true,"zx":0,"maxNumColors":256,"dither":1}' --avif '{"cqLevel":19,"cqAlphaLevel":17,"subsample":1,"tileColsLog2":0,"tileRowsLog2":0,"speed":6,"chromaDeltaQ":false,"sharpness":5,"denoiseLevel":0,"tune":0}' image-1.png image-2.png image-3.png
+```
+
+With optimized artwork checked into the repo, we could start loading them from HTML:
+
+```html
+<picture>
+  <source srcset="/day1/inner-frame.avif" type="image/avif">
+  <source srcset="/day1/inner-frame.webp" type="image/webp">
+  <img alt="" decoding="async" role="presentation" src="/day1/inner-frame.png">
+</picture>
+```
+
+It was repetitive writing the picture source code repeatedly, so we made an [Astro component](https://github.com/GoogleChromeLabs/designcember/blob/main/src/components/Pic/Pic.astro) to embed images with one line of code.
+
+```html
+<Pic filename="day1/inner-frame" role="presentation" />
+```
+
+## Screen reader and keyboard users
+
+Much of the experience of Designcember is through the art and interactive windows. It was important to us that a keyboard user could use the site and peek into windows, and that screen reader users get a nice narrated experience. 
+
+For example, when embedding the images we used `role="presentation"` to mark the image as presentational for screen readers. We felt that a user experience of between 5 and 12 fractured `alt` descriptions was going to be a poor experience. So, we marked the images as presentational and provided an overall window narration. Moving through the windows on a screen reader then has a nice narrative feeling, which we hoped would help deliver the whimsy and fun the site wants to share.
```

Comment:
This is really, really amazing. Honestly, it could be its own post / guide to help folks with image-heavy websites create a better accessible experience.

Should you want to move forward with such an idea, I'd gladly partner on writing it.